src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/MarshallingAttributeInfo.cs

```diff
@@ -90,6 +90,7 @@ public sealed record MarshalAsInfo(
         UnmanagedType UnmanagedType,
         CharEncoding CharEncoding) : MarshallingInfoStringSupport(CharEncoding)
     {
+        internal const UnmanagedType UnmanagedType_LPUTF8Str = (UnmanagedType)0x30;
```

Comment:
I assume the issue here is we target netstandard2.0 so must define it ourselves? If so, should we add a link to the official API we are following?

Reply:
```suggestion
        // UnmanagedType.LPUTF8Str is not in netstandard2.0, so we define a constant for the value here.
        // See https://docs.microsoft.com/dotnet/api/system.runtime.interopservices.unmanagedtype
        internal const UnmanagedType UnmanagedType_LPUTF8Str = (UnmanagedType)0x30;
```